pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v2/PersistentTopics.java

```diff
@@ -1456,7 +1460,9 @@ public void createSubscription(
             )
                     ResetCursorData resetCursorData,
             @ApiParam(value = "Is replicated required to perform this operation")
-            @QueryParam("replicated") boolean replicated
+            @QueryParam("replicated") boolean replicated,
+            @ApiParam(value = "JSON encoded list of properties for the Subscription")
+            @QueryParam("properties") String properties
```

Comment:
Should we put it in the body?  The url is too long and prone to truncation

Reply:
If we put this into the body then we will have more problems about compatibility, because unfortunately the body is already in use